packages/turbo-types/src/types/config-v1.ts

```diff
@@ -0,0 +1,301 @@
+   *
+   * @defaultValue `[]`
+   */
+  dependsOn?: Array<string>;
+
+  /**
+   * A list of environment variables that this task depends on.
+   *
+   * Note: If you are migrating from a turbo version 1.5 or below,
+   * you may be used to prefixing your variables with a $.
+   * You no longer need to use the $ prefix.
+   * (e.g. $GITHUB_TOKEN → GITHUB_TOKEN)
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#env
+   *
+   * @defaultValue `[]`
+   */
+  env?: Array<EnvWildcardV1>;
+
+  /**
+   * An allowlist of environment variables that should be made available in this
+   * task's environment, but should not contribute to the task's cache key,
+   * e.g. `AWS_SECRET_KEY`.
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#passthroughenv
+   *
+   * @defaultValue `null`
+   */
+  passThroughEnv?: null | Array<EnvWildcardV1>;
+
+  /**
+   * A priority-ordered (most-significant to least-significant) array of workspace-anchored
+   * Unix-style paths to `.env` files to include in the task hash.
+   *
+   * @defaultValue `null`
+   */
+  dotEnv?: null | Array<AnchoredUnixPath>;
+
+  /**
+   * The set of glob patterns indicating a task's cacheable filesystem outputs.
+   *
+   * Turborepo captures task logs for all tasks. This enables us to cache tasks whose runs
+   * produce no artifacts other than logs (such as linters). Logs are always treated as a
+   * cacheable artifact and never need to be specified.
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#outputs
+   *
+   * @defaultValue `[]`
+   */
+  outputs?: Array<string>;
+
+  /**
+   * Whether or not to cache the outputs of the task.
+   *
+   * Setting cache to false is useful for long-running "watch" or development mode tasks.
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#cache
+   *
+   * @defaultValue `true`
+   */
+  cache?: boolean;
+
+  /**
+   * The set of glob patterns to consider as inputs to this task.
+   *
+   * Changes to files covered by these globs will cause a cache miss and
+   * the task will be rerun.
+   *
+   * If a file has been changed that is **not** included in the set of globs,
+   * it will not cause a cache miss.
+   *
+   * If omitted or empty, all files in the package are considered as inputs.
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/configuration#inputs
+   *
+   * @defaultValue `[]`
+   */
+  inputs?: Array<string>;
+
+  /**
+   * Output mode for the task.
+   *
+   * "full": Displays all output
+   *
+   * "hash-only": Show only the hashes of the tasks
+   *
+   * "new-only": Only show output from cache misses
+   *
+   * "errors-only": Only show output from task failures
+   *
+   * "none": Hides all task output
+   *
+   * Documentation: https://turbo.build/repo/docs/reference/run#--output-logs-option
+   *
+   * @defaultValue `"full"`
+   */
+  outputLogs?: OutputModeV1;
```

Comment:
This was renamed in the v1 -> v2 migration
```suggestion
  outputMode?: OutputModeV1;
```

Reply:
fixed in 1c27fd3c2f779f39e0bd0a1c02fd2e4129d5d9ca